api/models/user_program.py

```diff
@@ -6,6 +6,10 @@
 class UserProgramQuery(BaseQuery):
     def upsert_user_program(self, user_id, data):
         user_program_details = self.get_latest_active_user_program(user_id)
+
+        if not user_program_details:
+            user_program_details = self.get_latest_user_program(user_id)
```

Comment:
in case there is no active user_program. As we have not yet decided how we will do resign up for these users. So currently not creating a new user program for completed users.